Source/WebCore/workers/shared/SharedWorkerObjectConnection.cpp

```diff
@@ -94,6 +94,15 @@ void SharedWorkerObjectConnection::postErrorToWorkerObject(SharedWorkerObjectIde
     ActiveDOMObject::queueTaskToDispatchEvent(*workerObject, TaskSource::DOMManipulation, WTFMove(event));
 }
 
+void SharedWorkerObjectConnection::reportNetworkUsageToWorkerObject(SharedWorkerObjectIdentifier sharedWorkerObjectIdentifier, size_t bytesTransferredOverNetworkDelta)
+{
+    ASSERT(isMainThread());
+    auto* workerObject = SharedWorker::fromIdentifier(sharedWorkerObjectIdentifier);
+    CONNECTION_RELEASE_LOG_ERROR("sendNetworkUsageToWorkerObject: sharedWorkerObjectIdentifier=%" PUBLIC_LOG_STRING ", worker=%p", sharedWorkerObjectIdentifier.toString().utf8().data(), workerObject);
```

Comment:
Seems like this should not be `RELEASE_LOG_ERROR`. I would suggest that it should either be a regular `RELEASE_LOG` that also logs `bytesTransferredOverNetworkDelta`, or should log nothing at all.